test-requirements.txt

```diff
@@ -1,17 +1,20 @@
-ansible-lint>=2.7.0,<3.0.0
-flake8==2.2.4
-hacking>=0.10.0,<0.11
-pep8==1.5.7
-pyflakes==0.8.1
-mccabe==0.2.1 # capped for flake8
+# The order of packages is significant, because pip processes them in the order
+# of appearance. Changing the order has an impact on the overall integration
+# process, which may cause wedges in the gate later.
 bashate>=0.2 # Apache-2.0
+flake8<2.6.0,>=2.5.4 # MIT
```

Comment:
Why are we capping flake8?

Reply:
This is a copy from what we're using elsewhere however we could remove this cap.